src/aoe_cut.jl

```diff
@@ -199,25 +200,24 @@ Get the survival fraction of a continuum after a AoE cut value `aoe_cut` for a g
 - `n_after`: Number of counts after the cut
 - `sf`: Survival fraction
 """
-function get_continuum_survival_fraction(aoe::Vector{<:Unitful.RealOrRealQuantity}, e::Vector{<:T}, center::T, window::T, aoe_cut::Unitful.RealOrRealQuantity,; inverted_mode::Bool=false, sigma_high_sided::Unitful.RealOrRealQuantity=Inf*unit(first(aoe))) where T<:Unitful.Energy{<:Real}
+function get_continuum_survival_fraction(aoe::Vector{<:Unitful.RealOrRealQuantity}, e::Vector{<:T}, center::T, window::T, aoe_cut::Unitful.RealOrRealQuantity,; inverted_mode::Bool=false, aoe_lowcut_vec::BitVector=falses(length(aoe)), sigma_high_sided::Unitful.RealOrRealQuantity=Inf*unit(first(aoe))) where T<:Unitful.Energy{<:Real}
     # scale unit
     e_unit = u"keV"
     # get energy around center
-    aoe = aoe[center - window .< e .< center + window]
-    e = e[center - window .< e .< center + window]
+    e_wdw = e[center - window .< e .< center + window]
     # get bin width
-    bin_width = get_friedman_diaconis_bin_width(e)
+    bin_width = get_friedman_diaconis_bin_width(e_wdw)
     # get number of events in window before cut
-    n_before = length(e)
+    n_before = length(e_wdw)
     # get energy after cuts
     e_survived, e_cut = if !inverted_mode
         #normal aoe version
         e[aoe_cut .< aoe .< sigma_high_sided], e[aoe .<= aoe_cut .|| aoe .>= sigma_high_sided]
     else
         #lq version
-        e[aoe .< aoe_cut .|| aoe .> sigma_high_sided], e[aoe_cut .<= aoe .<= sigma_high_sided]
+        e[aoe .< aoe_cut .&& .!aoe_lowcut_vec], e[aoe_cut .<= aoe .&& .!aoe_lowcut_vec]
     end
-    n_after = length(e_survived)
+    n_after = length(e_survived[center - window .< e_survived .< center + window])
```

Comment:
How about something like
```
e_wdw_idx = findall(center - window .< e .< center + window)
```
and then index arrays with `e_wdw_idx` where needed?